Source/OxyPlot/Rendering/ScreenPointHelper.cs

```diff
@@ -22,11 +24,17 @@ public static class ScreenPointHelper
         /// <param name="point">The point.</param>
         /// <param name="points">The points.</param>
         /// <returns>The nearest point.</returns>
+        /// <exception cref="ArgumentNullException">The <paramref name="points"/> is <c>null</c>.</exception>
         public static ScreenPoint FindNearestPointOnPolyline(ScreenPoint point, IList<ScreenPoint> points)
         {
             double minimumDistance = double.MaxValue;
             var nearestPoint = default(ScreenPoint);
 
+            if (points == null)
```

Comment:
sorry for commenting on the wrong place, I am still learning... thanks for the correction! :-)